openpower/configs/p9dsu_defconfig

```diff
@@ -44,7 +44,7 @@ BR2_OPENPOWER_PLATFORM=y
 BR2_OPENPOWER_POWER9=y
 BR2_HOSTBOOT_CONFIG_FILE="p9dsu.config"
 BR2_OPENPOWER_MACHINE_XML_GITHUB_PROJECT_VALUE="p9dsu-xml"
-BR2_OPENPOWER_MACHINE_XML_VERSION="32ce616ef80f5cc3f128b3aa08fe50954a087888"
```

Comment:
This is what's currently in op-build master, and the bump to 734a35ebe5af49dba has *not* even been in a pull request for op-build master yet.

Please ensure that this change is merged to master *first*, and then cherry-picked back to 2.0.y and 2.3.y branches. If for some reason (and I can't think of any) these do not apply for master, please explain *why* that's the case in the commit message here.

Reply:
@stewart-ibm 
1)  There is no plan to support Boston in the master branch at this time.
2)  I have noticed that beside BR2_OPENPOWER_MACHINE_XML_VERSION variable change, the xml file is different between master branch and 2.0.y and 2.3.y.   I don't feel like we need to make this move to a shipped firmware.

diff 2.0.y  master
3d2
< BR2_PACKAGE_HOST_LINUX_HEADERS_CUSTOM_4_19=y
14a14
> BR2_ROOTFS_USERS_TABLES="$(BR2_EXTERNAL_OP_BUILD_PATH)/configs/users-table"
19c19
< BR2_LINUX_KERNEL_CUSTOM_VERSION_VALUE="4.19.30"
---
> BR2_LINUX_KERNEL_CUSTOM_VERSION_VALUE="5.1.4"
40a41,42
> BR2_PACKAGE_SUDO=y
> BR2_PACKAGE_UTIL_LINUX_AGETTY=y

diff 2.3.y master
19c19
< BR2_LINUX_KERNEL_CUSTOM_VERSION_VALUE="5.0.7"
---
> BR2_LINUX_KERNEL_CUSTOM_VERSION_VALUE="5.1.4"

It appears that kernel 5.1.4 is not merged to 2.3.y and therefore 2.3.y cannot use xml file from master.

Reply:
If Boston cannot be supported, then we need to:
1) Remove it from master
2) find replacement machines for the Bostons we have and rely on for our work and regression testing.

Reply:
You also do not need the same kernel version in 2.3.y and master, you just need to resolve any conflicts.

Reply:
Also, as per https://open-power.github.io/op-build/process/KernelTree.html#versioning op-build 2.0.y will *never* get any kernel later than a 4.19 kernel (it's a longterm kernel).

What *exactly* is the dependency between kernel version and the p9dsu-xml change? I can't see how there could possibly be one.